app/views/requests/show.html.erb

```diff
@@ -49,14 +48,18 @@
       <div>
         <h3>Pending inbox messages: <%= @conversation_count%></h3>
       </div>
-
     <% end %>
 
-
-
     <div class="request-show-buttons">
-    <a href="#" class="btn btn-primary edit-btn">Edit</a>
-    <%= link_to "Cancel", request_path, method: :delete, data: { confirm: "Are you sure?" }, class: "btn btn-outline-danger" %>
+      <% if @request.creator == current_user %>
+        <a href="#" class="btn btn-primary edit-btn">Edit</a>
+        <%= link_to "Cancel", request_path, method: :delete, data: { confirm: "Are you sure?" }, class: "btn btn-outline-danger" %>
+      <% else %>
+         <p>This request is already confirmed by a helpal.</p>
```

Comment:
We'll need to think of how we want this to work. Showing that text to a user doesn't make sense given the request is still in a pending state and has not been confirmed by another user. It might be simpler to just show no text.